lib/features/children/overview/pages/children_overview_page.dart

```diff
@@ -0,0 +1,111 @@
+        width: double.infinity,
+        height: double.infinity,
+        // color: Colors.amber,
+        child: Column(
+          children: [
+            SvgPicture.asset(
+              'assets/images/logo_blue.svg',
+              height: size.height * 0.035,
+            ),
+            Container(
+              padding: const EdgeInsets.only(left: 20, right: 20, top: 20),
+              height: size.height * 0.921,
+              child: BlocConsumer<ChildrenOverviewCubit, ChildrenOverviewState>(
+                listener: (context, state) {
+                  log('children overview state changed on $state');
+                  if (state is ChildrenOverviewErrorState) {
+                    ScaffoldMessenger.of(context).showSnackBar(
+                      SnackBar(
+                        content: Text(
+                          state.errorMessage,
+                          textAlign: TextAlign.center,
+                        ),
+                        backgroundColor: Theme.of(context).errorColor,
+                      ),
+                    );
+                  }
+                },
+                builder: (context, state) {
+                  if (state is ChildrenOverviewLoadingState) {
+                    return const Center(child: CircularProgressIndicator());
+                  } else if (state is ChildrenOverviewUpdatedState) {
+                    final user = context.read<AuthCubit>().state.user;
+                    final currency = NumberFormat.simpleCurrency(
+                      name: Util.getCurrencyName(
+                        country: Country.fromCode(user.country),
+                      ),
+                    );
+
+                    return SingleChildScrollView(
+                      child: Column(
+                        children: state.profiles
+                            .map(
+                              (profile) => ChildItem(
+                                name: profile.firstName,
+                                total: profile.wallet.total,
+                                currencySymbol: currency.currencySymbol,
+                                pending: profile.wallet.pending,
+                              ),
+                            )
+                            .toList(),
+                      ),
+                    );
+                  } else {
+                    return Container();
+                  }
+                },
+              ),
+            ),
+          ],
+        ),
+      ),
+      floatingActionButtonLocation: FloatingActionButtonLocation.centerFloat,
+      floatingActionButton: Padding(
+        padding: const EdgeInsets.only(left: 35, right: 35, bottom: 30),
+        child: ElevatedButton(
+          onPressed: () {
+            if (context.read<VPCCubit>().vpcGained) {
+              context.goNamed(Pages.createChild.name);
+            } else {
+              context.goNamed(Pages.giveVPC.name);
+            }
```

Comment:
I'm an old school guy in this case)). According to Steve McConnell, it's better to make 'else' statements more explicit. Would be great to talk about it somewhere in the future)

Reply:
Got it, I am on the other hand always trying to avoid unnecessary nesting or statements. As this practice from my experience can lead to confusions and big if else statements